config/model_configs/amip_diagedmf.yml

```diff
@@ -0,0 +1,15 @@
+anim: false
+atmos_config_file: "config/gpu_configs/gpu_aquaplanet_diagedmf.yml"
```

Comment:
This is using a config file from atmos. It's duplicated in our `config/atmos_configs/gpu_aquaplanet_diagedmf.yml`, but maybe it would be better for this to use the version that's in ClimaCoupler, so that even if atmos changes their setup, we continue to compare analogous atmos/coupled simulations

Reply:
I could copy [this line](https://github.com/CliMA/ClimaCoupler.jl/blob/69a396915161bea19c0abe6919d7be0e3062e3f8/experiments/AMIP/components/atmosphere/climaatmos.jl#L252) to use ClimaCoupler instead of ClimaAtmos, and keep support for both options. I'm not sure of the best way to do this - maybe we add another option to the coupler configs (something like `coupler_atmos_config_file`, but that might be confusing)

Reply:
That makes sense! I added a flag that's only used by the coupled benchmark runs, which specifies that we want to use an atmos config file from ClimaCoupler instead of ClimaAtmos